scripts/check_frontend_coverage.py

```diff
@@ -516,59 +568,67 @@ def get_stanzas_from_lcov_file():
     return stanzas_list
 
 
-def check_fully_covered_filenames_list_is_sorted():
-    """Check if FULLY_COVERED_FILENAMES list is in alphabetical order."""
-    if FULLY_COVERED_FILENAMES != sorted(
-            FULLY_COVERED_FILENAMES, key=lambda s: s.lower()):
-        sys.exit('The \033[1mFULLY_COVERED_FILENAMES\033[0m list must be kept'
-                 ' in alphabetical order.')
+def check_not_fully_covered_filenames_list_is_sorted():
+    """Check if NOT_FULLY_COVERED_FILENAMES list is in alphabetical order."""
+    if NOT_FULLY_COVERED_FILENAMES != sorted(
+            NOT_FULLY_COVERED_FILENAMES, key=lambda s: s.lower()):
+        sys.exit(
+            'The \033[1mNOT_FULLY_COVERED_FILENAMES\033[0m list must be'
+            ' kept in alphabetical order.')
 
 
 def check_coverage_changes():
-    """Checks if the whitelist for fully covered files needs to be changed by:
-    - New file insertion
+    """Checks if the blacklist for not fully covered files needs to be changed
+    by:
     - File renaming
     - File deletion
 
     Raises:
         Exception: LCOV_FILE_PATH doesn't exist.
     """
     if not os.path.exists(LCOV_FILE_PATH):
-        raise Exception('Expected lcov file to be available at {}, but the'
-                        ' file does not exist.'.format(LCOV_FILE_PATH))
+        raise Exception(
+            'Expected lcov file to be available at {}, but the'
+            ' file does not exist.'.format(LCOV_FILE_PATH))
 
     stanzas = get_stanzas_from_lcov_file()
-    whitelist = list(FULLY_COVERED_FILENAMES)
+    remaining_blacklisted_files = list(NOT_FULLY_COVERED_FILENAMES)
     errors = ''
 
     for stanza in stanzas:
         file_name = stanza.file_name
         total_lines = stanza.total_lines
         covered_lines = stanza.covered_lines
 
-        if file_name in whitelist:
+        if file_name not in remaining_blacklisted_files:
             if total_lines != covered_lines:
-                errors += ('\033[1m{}\033[0m file is in the whitelist but its'
-                           ' coverage decreased. Make sure it is fully covered'
-                           ' by Karma unit tests.\n'.format(file_name))
-
-            whitelist.remove(file_name)
+                errors += (
+                    '\033[1m{}\033[0m seems to be not completely tested.'
+                    ' Make sure it\'s fully covered before pushing'
+                    ' your changes.\n'.format(file_name))
```

Comment:
Maybe drop the "before pushing your changes" part of the message as this can run on CI too...

Reply:
Done.